rita/src/rita_common/http_client/mod.rs

```diff
@@ -73,7 +76,16 @@ impl Handler<Hello> for HTTPSyncExecutor {
 
     fn handle(&mut self, msg: Hello, _: &mut Self::Context) -> Self::Result {
         info!("sending {:?}", msg);
-        let my_id = serde_json::to_string(&msg.my_id)?;
+
+        let my_id = if SETTING.get_future() {
```

Comment:
I don't think this works, this says that we take either or not that we can take both?

Reply:
what if half the nodes have updated and the other half have not?